src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -29,6 +29,7 @@ private class Tagger : ForegroundThreadAffinitizedObject, IAccurateTagger<IClass
             private readonly SemanticClassificationBufferTaggerProvider _owner;
             private readonly ITextBuffer _subjectBuffer;
             private readonly ITaggerEventSource _eventSource;
+            private readonly SemanticClassifier _classifier;
```

Comment:
changed from static helpers to an instance type now that it contains more interesting logic around caching/shutdown.